src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/ContestedCourtHelper.java

```diff
@@ -320,6 +321,111 @@ private static String getMidlandInterimHearingFRC(Map<String, Object> interimHea
         return EMPTY;
     }
 
+    public static String getSelectedHearingFrc(Map<String, Object> hearingData) {
```

Comment:
Shouldn't this section be in the consented hearing helper? Might help make it easier to understand why there's duplicate code and when to use the correct getSelectedHearingFrc method